Saule/Queries/Filtering/FilterProperty.cs

```diff
@@ -9,11 +13,34 @@ public class FilterProperty
         /// Initializes a new instance of the <see cref="FilterProperty"/> class.
         /// </summary>
         /// <param name="name">property name</param>
-        /// <param name="value">property value</param>
-        public FilterProperty(string name, string value)
+        /// <param name="values">property values in one string in csv notation</param>
+        public FilterProperty(string name, string values)
         {
-            Value = value;
             Name = name.ToPascalCase();
+            Value = values;
+
+            // Spliting the string into multiple values with csv notation
+            Values = new List<string>();
+
+            var match = new Regex("(?:,\"|^\")(\"\"|[\\w\\W]*?)(?=\",|\"$)|(?:,(?!\")|^(?!\"))([^,]*?)(?=$|,)").Matches(values);
```

Comment:
Can you maybe put a comment that explains this regex? Some time in the future we may want to remember how it works.

Reply:
I tried my best to explain this one. I hope that it's enough.